README.md

```diff
@@ -1823,7 +1823,7 @@ Sets the list of resources to look for when a client requests an index of the di
 
 ##### `docroot`
 
-**Required**. Sets the [`DocumentRoot`][] location, from which Apache serves files.
```

Comment:
@robbat2 this is interesting, and surprising.
`undef` usually erases the value, taking on whatever is default, i didn't expect this outcome… cc / @hlindberg